### PR TITLE
Fast Fault Mirror Coating Update; read from JSON Doc

### PR DESCRIPTION
**Pull request by nrwslac**
## Description

- Remove hardcoded beam parameters in `P_StripeProtections`
- Running lcls-twincat-optics from dev version 0.0.0
- Running lcls-twincat-motion from dev version 0.0.0
- Uses upgraded `FB_MirrorTwoCoatingProtection` which loads beam parameters from JSON Doc
- Pass `MOTION_GVL.fbStandardPMPSDB.bReadPmpsDb` to `FB_MirrorTwoCoatingProtection` calls.

## Motivation and Context

PMPS was upgraded to load state beam parameters from the PMPS Database, but fast fault hardcoded beam parameters need to read from the PMPS Database as well. 
## How Has This Been Tested?

- lcls-twincat-optics and lcls-twincat-motion 0.0.0 loaded onto the this project. 3 functions of type FB_MirrorTwoCoatingProtection were declared and called.
toggled `bRefresh` in `FB_Standard_PMPSDB` call inside `PRG_PMPS` and checked that each nRefreshAttempts incremented.
- Checked array elements for `bBeamParamsLoaded` True.

https://github.com/pcdshub/lcls-twincat-optics/pull/18

## Where Has This Been Documented?

- Request to remove hardcoded beam parameters here: https://confluence.slac.stanford.edu/pages/viewpage.action?pageId=368318178
- Should document the addition of global refresh declaration and write in the somewhere if this solution is accepted.
## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive comments
- [ ] Test suite passes locally
- [ ] Libraries are set to fixed versions and not ``Always Newest``
- [ ] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

File: lcls-plc-tmo-optics/tmo_optics/POUs/MAIN.TcPOU [hunk 1]
```diff
@@ -958,7 +958,7 @@ END_VAR]]></Declaration>
     PRG_States();
 // PMPS
     PRG_PMPS();
-    P_StripeProtections();
+    //P_StripeProtections();
```
Comment (ZLLentz) [inline lcls-plc-tmo-optics/tmo_optics/POUs/MAIN.TcPOU:961]: If you've moved this around, you may consider deleting the line so nobody accidentally uncomments it and runs the stripe protection program twice.

**Review by ZLLentz — COMMENTED** [review]
This looks reasonable to me. I'm holding off on the approval until it has numbered release references but otherwise LGTM :+1:

File: lcls-plc-tmo-optics/tmo_optics/POUs/MAIN.TcPOU (hunk 1 above)
Comment (nrwslac) [reply lcls-plc-tmo-optics/tmo_optics/POUs/MAIN.TcPOU:961]: removed.

**Comment by nrwslac** [comment]
> This looks reasonable to me. I'm holding off on the approval until it has numbered release references but otherwise LGTM +1

This set to released versions of optics and motion now.

**Review by ZLLentz — APPROVED** [review]
Looks good to me!

**Pull request merged**